### PR TITLE
AG-12052 rework get started

### PR DESCRIPTION
**Pull request by AbiGlazier**

### CODE REVIEW

File: packages/ag-charts-website/src/content/docs/quick-start/index.mdoc [hunk 1]
```diff
@@ -219,26 +266,25 @@ root.render(<ChartExample />);
...
   selector: 'app-root',
   standalone: true,
   imports: [AgCharts],
   // ag-charts-angular component with chartOptions attribute
-  template:
-  `<ag-charts
-    style="height: 100%"
-    [options]="chartOptions">
-   </ag-charts>`,
+  template:``,
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/docs/quick-start/index.mdoc:289]: Nit: Might be better to just drop this from the example, or add some indicator that the example is glossing over this part?

e.g.
```
template: `<!-- ... -->`
```

File: packages/ag-charts-website/src/content/docs/quick-start/index.mdoc [hunk 2]
```diff
@@ -288,62 +347,82 @@ Replace your `app.vue` (or root component) with the following code:
...
+{% note %}
+To live-edit the code, open the example in CodeSandbox or Plunker using the buttons to the lower-right.
+{% /note %}
+
 ## Next Steps
 
-{% if isFramework("react") %}
+-   Read our [Introductory Tutorial](./create-a-basic-chart/).
+
+<!-- {% if isFramework("react") %}
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/docs/quick-start/index.mdoc]: Nit: Should this be commented?

File: packages/ag-charts-website/src/content/docs/quick-start/index.mdoc [hunk 3]
```diff
@@ -365,4 +444,4 @@ When you run your app, you should see a very simple Vue chart:
 
 -   [How to Create a Basic JavaScript Chart](./create-a-basic-chart/)
 
-{% /if %}
+{% /if %} -->
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/docs/quick-start/index.mdoc]: Ditto

File: packages/ag-charts-website/src/content/docs/quick-start/index.mdoc (hunk 1 above)
Comment (AbiGlazier) [reply packages/ag-charts-website/src/content/docs/quick-start/index.mdoc:289]: I've removed the comment from above 'ag-charts-angular component with chartOptions attribute' but left the empty template in in line with the AG Grid angular getting started guide: https://www.ag-grid.com/angular-data-grid/getting-started/ 

Let me know if you're happy with this approach.

File: packages/ag-charts-website/src/content/docs/quick-start/index.mdoc (hunk 2 above)
Comment (AbiGlazier) [reply packages/ag-charts-website/src/content/docs/quick-start/index.mdoc]: Removed

File: packages/ag-charts-website/src/content/docs/quick-start/index.mdoc (hunk 3 above)
Comment (AbiGlazier) [reply packages/ag-charts-website/src/content/docs/quick-start/index.mdoc]: Removed

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**